src/app/containers/concerts-container/concerts-container.component.ts

```diff
@@ -14,18 +16,14 @@ import { ConcertPreviousResponse, PreviousConcertsService } from '../../services
 export class ConcertsContainerComponent implements OnInit {
   private currentPlayedElement?: HTMLAudioElement;
 
-  isConcertPreviousItemsExist = true;
-
-  concertsPreviousItems: readonly ConcertPreviousItem[] = [];
+  concertsPreviousItems$!: Observable<readonly ConcertPreviousItem[]>;
 
   constructor(private readonly previousConcertsService: PreviousConcertsService, public dialog: MatDialog) {}
 
   ngOnInit(): void {
-    this.previousConcertsService.getPreviousConcertsData().subscribe((response: readonly ConcertPreviousResponse[]) => {
-      this.concertsPreviousItems = mapConcertPreviousResponseToConcertPreviousItems(response);
-    }, (() => {
-      this.isConcertPreviousItemsExist = false;
-    }));
+    this.concertsPreviousItems$ = this.previousConcertsService
+      .getPreviousConcertsData()
+      .pipe(map((response: readonly ConcertPreviousResponse[]) => mapConcertPreviousResponseToConcertPreviousItems(response)));
```

Comment:
itt lehet annyi is eleg, hogy `.pipe(map(mapConcertPreviousResponseToConcertPreviousItems));` mert kb ugyan azt jelenti (de csak tipp)